docs/admin/runai-setup/cluster-setup/cluster-install.md

```diff
@@ -27,6 +26,34 @@ On the next page:
 * (SaaS and remote self-hosted cluster only) Install a trusted certificate to the domain entered above.
 *  Run the [Helm](https://helm.sh/docs/intro/install/) command provided in the wizard.
 
+## Cluster Table
+
+After you have installed your cluster on the platform, you will see it appear in the *Cluster Table*. The *Cluster Table* provides a quick and easy way to see the status of your cluster.
+
+In the left menu, press *Clusters* to view the cluster table. Use *Add filter* to add one or more filter results based on the columns that are in the table. In the *Contains* pane, you can use partial or complete text. Filtered text is ***not*** case sensitive. To remove the filter, press *X* next to the filter.
+
+The table provides the following columns:
+
+* **Cluster**&mdash;the name of the cluster.
+* **Status**&mdash;the status of the cluster. For more information see [Cluster status](#cluster-status). Hover over the information icon to see a short description and links to troubleshooting.
+* **Creattion time**&mdash;the timestamp the cluster was created.
+* **URL**&mdash;the URL that was given to the cluster at the time of creation.
+* **Run:ai cluster version**&mdash;the version of the cluster.
+* **Run:ai cluster UUI**&mdash;the unique ID of the cluster.
+
+### Cluster Status
+
+The following table describes the different statuses that a cluster could be in.
+
+| Status | Description |
+| -- | -- |
+| Waiting to connect | The cluster is waiting to connect or has never been connected. |
+| Disconnected | All the services that communicate with the control plane are not reporting. This may bean issue outside of of the Run:ai platform. |
```

Comment:
```suggestion
| Disconnected | There is no communication from the cluster to the Control Plane. This may be a network issue outside of the Run:ai platform. |
```